utility/src/test/java/org/oppia/util/logging/ExceptionsHelperTest.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.util.logging
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.model.ExceptionLog
+import org.robolectric.annotation.Config
+import javax.inject.Singleton
+
+@RunWith(AndroidJUnit4::class)
+@Config(manifest = Config.NONE)
+class ExceptionsHelperTest {
+
+  private val exceptionLog = ExceptionLog.newBuilder().setMessage("TEST").build()
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
```

Comment:
Can you add tests for the different if conditions you have written above? 
e.g.
* When cause is empty and stack trace is 0
* When cause is empty and stark trace is > 0
* When exception message is null

Reply:
File removed.